macos/FluentUI/Separator.swift

```diff
@@ -77,17 +75,4 @@ open class Separator: NSView {
 			layer.backgroundColor = style.color.cgColor
 		}
 	}
-	
-	open override func isAccessibilityElement() -> Bool {
-		return false
-	}
-
-	open override var intrinsicContentSize: CGSize {
-		switch orientation {
-		case .horizontal:
-			return CGSize(width: NSView.noIntrinsicMetric, height: frame.height)
-		case .vertical:
-			return CGSize(width: frame.width, height: NSView.noIntrinsicMetric)
-		}
-	}
```

Comment:
We had this on the iOS control. I'm not sure the benefit of specifying `NSView.noIntrinsicMetric`, so I removed it. Would anyone have insight on what it is useful for?

Reply:
I found this within the documentation for `AppKit.NSLayoutConstraint`

"Note that not all views have an intrinsicContentSize.  A horizontal slider has an intrinsic height, but no intrinsic width - the slider artwork has no intrinsic best width.  A horizontal NSSlider returns (NSViewNoIntrinsicMetric, <slider height>) for intrinsicContentSize.  An NSBox returns (NSViewNoIntrinsicMetric, NSViewNoIntrinsicMetric).  The _intrinsic_ content size is concerned only with data that is in the view itself, not in other views."